submissions/dafen173/js-dom/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>DOM task</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <meta name="viewport" content="width=device-width, intial-scale=1.0">
+</head>
+<body>
+    
+    <div class="maincontainer">
+        <div class="dropdown">
+          <div class="mainmenubtn" id='menubtn'>Типовые инсталляции</div>
+          <div class="dropdown-child" id='dropchild'>
+            <div class='installations' id="section1">6500 ANSI Lm на улице</div>
+            <div class='installations' id="section2">Сшивка двух короткофокусных проекторов</div>
+            <div class='installations' id="section3">Спарка двух проекторов по 6500 ANSI Lm</div>
+            <div class='installations' id="section4">Видеостена на базе трех проекторов</div>
+          </div>
+        </div>  
+        <div class="content">
+          <h1 class='headline'>6500 ANSI Lm на улице</h1>
+          <div class="container">
+              Работает проектор типа HITACHI CP-WX8265 3LCD  Lamp  1280х800   WXGA 6500lm  3600уе на экран шириной 3 м.
+          </div>
+          <img class='pic' src='img/first.png' alt="Фото инсталляции"> 
+          <br>
+          <img class='pic2' src='img/0697.png' alt="">           
+        </div>
+    </div>
+    
```

Comment:
Remove unused empty lines.

Reply:
done

https://github.com/dafen173/kottans_frontend/blob/master/task_js_dom/index.html

Reply:
ok?

Reply:
Fix all issues and then push the commit. The current pull request will be updated automatically. And then we review your changes.)

Reply:
ok, thanks

Reply:
> Fix all issues and then push the commit. The current pull request will be updated automatically. And then we review your changes.)

Done. Please, review.

Reply:
Thanks!